Common/TDSQASystemAPI/BL/testpackage/administration/ItembankConfigurationDataService.cs

```diff
@@ -21,6 +21,7 @@ public class ItembankConfigurationDataService : IItembankConfigurationDataServic
         private const string GRADE_PROP_NAME = "Grade";
 
         private readonly IItembankConfigurationDataQueryService itembankConfigurationDataQueryService;
+        private readonly ITestPackageDao<ClientDTO> clientDao;

```

Comment:
Turns out I needed this after all.  Initially, I created the `IItembankConfigurationDataQueryService` to handle all the finds/lookups.  I'm not sure if `IItembankConfigurationDataQueryService` provides much value, but it was too risky at this point to remove it.